CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file.
 
+## [v3.3.0](https://github.com/voxpupuli/puppet-syntax/tree/v3.3.0) (2023-02-02)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-syntax/compare/v3.2.1...v3.3.0)
+
+**Merged pull requests:**
+
+- Pull in Puppet 6.29 in CI [\#135](https://github.com/voxpupuli/puppet-syntax/pull/135) ([ekohl](https://github.com/ekohl))
+- Lazy load required classes [\#134](https://github.com/voxpupuli/puppet-syntax/pull/134) ([ekohl](https://github.com/ekohl))
+
+## [v3.2.1](https://github.com/voxpupuli/puppet-syntax/tree/v3.2.1) (2022-05-16)
```

Comment:
No changes in 3.2.1?

Reply:
https://github.com/voxpupuli/puppet-syntax/compare/v3.2.0...v3.2.1 I think we did 3.2.1 because 3.2.0 had something missing in the changelog?